string_utils/fms_string_utils.F90

```diff
@@ -112,11 +114,20 @@ subroutine c_free(ptr) bind(c,name="free")
   module procedure cpointer_fortran_conversion
 end interface
 
-!> Converts a number to a string
+!> Converts a number or a Boolean value to a string
 !> @ingroup fms_mod
 interface string
-   module procedure string_from_integer
-   module procedure string_from_real
+  module procedure string_from_logical
+  module procedure string_from_integer
+  module procedure string_from_r4, string_from_r8
+end interface
```

Comment:
Will the conflict with 2023.01-beta3?

Reply:
Updated `fms_string_utils_mod` merged from main in 42c2d1c1caa18fb9854319c541a4b315ffa61c64.